### DISCUSSION TITLE
Trouble Setting Up RavenDB on Coolify with Let's Encrypt Security Mode

### DISCUSSION THREAD

**Discussion by SteffenGottschalk** [ravendb/ravendb#18677] [Q&A] [answered]
Hi everyone,

I'm having trouble setting up RavenDB on Coolify, specifically in Let's Encrypt security mode. During the setup, I encountered the following error message:

```vbnet
Setting up RavenDB in Let's Encrypt security mode failed.
System.InvalidOperationException: Setting up RavenDB in Let's Encrypt security mode failed.
 ---> System.InvalidOperationException: Validation failed.
 ---> System.InvalidOperationException: Failed to simulate running the server with the supplied settings using: https://a.xxxxxxx.ravendb.community
 ---> System.InvalidOperationException: Failed to start WebHost on node 'A'. The specified ip address might not be reachable due to network issues. It can happen if port '443' is not allowed for the non-root RavenDB process. Try using setcap to allow it: sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/lib/ravendb/server/Raven.Server
```

I've checked my network settings, and port 443 should be accessible. I'm not sure how to proceed with the suggested setcap command or if there's something else I should check or configure.

Has anyone encountered a similar issue or have any advice on how to resolve this?

Thanks in advance for your help!

Best regards,
Steffen

**Comment by ayende** [comment, marked as answer]
```
sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/lib/ravendb/server/Raven.Server
```

Please run this command on your instance?
The issue is likely that the process doesn't have permissions to access ports < 1024
